command/meta.go

```diff
@@ -160,7 +161,7 @@ func (m *ApiMeta) Process(args []string) ([]string, error) {
 	cfg := &gate.Configuration{
 		BasePath:      m.gateEndpoint,
 		DefaultHeader: make(map[string]string),
-		UserAgent:     "Spin CLI version", // TODO(jacobkiefer): Add a reasonable UserAgent.
+		UserAgent:     fmt.Sprintf("%s: %s", version.Header, version.VersionString()),
```

Comment:
sorry, not familiar with programming model here and github UI isnt helpful.
Is the UserAgent the HTTP header? It should be in the format "<program>/<version> ..."
I'm not sure what version.Header is here. The name "header" is odd.

Reply:
Not tied to the name. Is the "/..." common? Something like `/Spin CLI: <version>`?